webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -35,6 +35,23 @@ const myStatusClasses: Record<string, string> = {
     [myStatus.DONE]: 'ticket-status--done',
 };
 
+function preProcessDescription(description: string): string {
```

Comment:
Cant we do it on backend side? As the data is coming from there only. We already have a function there to handle all this. Also we are supporting just links here, we need to handle other cases as well